autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/on_server_notif.rs

```diff
@@ -116,16 +115,24 @@ impl WebPushClient {
 
         // Filter out TTL expired messages
         let now_sec = sec_since_epoch();
-        messages.retain(|n| {
-            if !n.expired(now_sec) {
+        // Topic messages require immediate deletion from the db
+        let mut expired_topic_sort_keys = vec![];
+        messages.retain(|msg| {
+            if !msg.expired(now_sec) {
                 return true;
             }
-            // TODO: A batch remove_messages would be nicer
-            if n.sortkey_timestamp.is_none() {
-                self.spawn_remove_message(n.sort_key());
+            if msg.sortkey_timestamp.is_none() {
+                expired_topic_sort_keys.push(msg.sort_key());
             }
             false
         });
+        // TODO: A batch remove_messages would be nicer
```

Comment:
technically, we should use the Topic Family to say that only one version of this should be stored or returned, but we can work on that later.